aggregators/1.0/modules/ROOT/pages/aggregators-module-reference.adoc

```diff
@@ -182,185 +175,163 @@ image::time-based-aggregator.png[Time based Aggregator]
+* Group timeout +
+When a group has to be released because all the necessary elements for the group did not arrive within the expected time. If a group has timed out but is not yet evicted, it will reject attempts to add any new values to that group.
 
-Lastly, when elements that reach group-based aggregators come from a sequence that was splitted (by a xref:mule-runtime::for-each-scope-concept.adoc[ForEach] component for example), each will have assigned a different `sequenceNumber`. In that case, they are sorted in increasing order prior to the aggregation release.
+* Group eviction +
+When a group is removed from the aggregator, regardless of whether it was completed or timed out. If a new element with that group's ID is received by the aggregator, the group will be created again.
+
+Lastly, when elements that reach group-based aggregators come from a sequence that is splitted (by a xref:mule-runtime::for-each-scope-concept.adoc[ForEach] component for example), each elements gets assigned a different `sequenceNumber`. In that case, the elements are sorted in increasing order prior to the aggregation release.
 
 ==== Parameters
 
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Name | String | The name of the aggregator. It can later be used to be referenced by an aggregator listener || x
+| Name | String | The name of the aggregator. n aggregator listener can later reference the scope by name. || x
 | Content | Expression | The expression that defines what to aggregate. The result of the evaluation is the value stored in the aggregation. | #[payload] |
-| Group Id | Expression | The expression to be evaluated for every new message received in order to get the ID for the group where it should be aggregated. | #[correlationId] |
-| Group Size | Number | The maximum size to assign to the group with the group ID resolved. All messages with the same group ID must have the same group size. If not, only the first resolved group size will be considered correct, and a warning will be logged for every one that does not match it. | #[itemSequenceInfo.sequenceSize] |
-| Eviction Time | Number | The time to remember a group ID once it was completed or timed out (0 means: don't remember, -1: remember forever) | 180 |
+| Group Id | Expression | The expression to evaluate for every new message received in order to get the ID for the group where it should be aggregated. | #[correlationId] |
+| Group Size | Number | The maximum size to assign to the group with the group ID resolved. All messages with the same group ID must have the same group size. If not, only the first resolved group size is considered correct. A warning is logged for every one that does not match it. | #[itemSequenceInfo.sequenceSize] |
+| Eviction Time | Number | The time to remember a group ID once it is completed or timed out (0 means: don't remember, -1: remember forever) | 180 |
```

Comment:
```suggestion
| Eviction Time | Number | The time to remember a group ID after it is completed or timed out (0 means: don't remember, -1: remember forever). | 180 |
```

Reply:
But...is it correct to say that a group ID is "completed" or "timed out"? How is an ID complete or timed out?
Do we mean that the group ID is either no longer used or expired?